big-red-shuttle/Network.swift

```diff
@@ -23,17 +27,17 @@ class System {
     }
     
     func userLocation() -> Coordinate? {
-       return Location.sharedLocation.currentUserLocation
+       return Location.shared.currentUserLocation
     }
 
     func busLocation() -> Coordinate? {
-        return Location.sharedLocation.currentBusLocation
+        return Location.shared.currentBusLocation
```

Comment:
These are better suited as properties instead of functions, though I'm not sure I agree with the excessive abstraction into this System class thing.